test/unit-test/test_lib/test_analysis/test_user.py

```diff
@@ -1,34 +1,135 @@
 import unittest
 from lib.analysis import user
+import lib.config as config
+from lib.in_out import reader, saver
+import lib.nickTracker as nickTracker
+from mock import patch
+import networkx as nx
 import lib.util as util
 import os
+import StringIO
+import sys
+import filecmp
 
 current_directory = os.path.dirname(os.path.realpath(__file__))
+state0 = 0
```

Comment:
please change this variable name to probably something like build_graphs_util_call_counter or something short. state0 seems vague.